### PR TITLE
Clear status messages when route changed

### PR DESCRIPTION
**Pull request by ekaragodin**

### CODE REVIEW

File: src/App/__tests__/App.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright 1999-2018. Plesk International GmbH. All rights reserved.
+
+import { createElement } from '@plesk/ui-library';
+import App from '../App';
+import { mount } from 'enzyme';
+
+describe('App', () => {
+    it('renders correctly', () => {
+        mount(
```
Comment (pabla) [inline src/App/__tests__/App.test.js]: Maybe, is it better to use shallow?

Comment (ekaragodin) [reply src/App/__tests__/App.test.js]: I agree, done.

File: src/Route/Route.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import { Route as ReactRoute } from 'react-router-dom';
+
+const stripTags = str => str.replace(/<\w+(\s+("[^"]*"|'[^']*'|[^>])+)?(\/)?>|<\/\w+>/gi, '');
+
+const renderTitle = (title = '') => {
+    const parts = document.title.split(' - ');
+
+    const documentTitle = document.querySelector('title');
+    if (documentTitle) {
+        documentTitle.update(
```
[the remark this answers is not in the document]
Comment (ekaragodin) [reply src/Route/Route.js]: Fixed.

File: src/ServerForm/ServerForm.js [hunk 3]
```diff
@@ -73,27 +79,33 @@ class ServerForm extends Component {
...
-                    state: null,
-                    errors,
-                }, () => {
-                    const { successUrl, history, onSuccess } = this.props;
+            const newState = {
+                state: null,
+            };
+
+            if (Array.isArray(errors)) {
+                errors.map(message => statusMessages.add({ intent: 'danger', message }));
```
[the remark this answers is not in the document]
Comment (ekaragodin) [reply src/ServerForm/ServerForm.js]: Fixed.

**Pull request merged**